### PR TITLE
Add IRC landing page

### PR DESCRIPTION
**Pull request by baude**
Add a simple irc landing page for connecting to Libera.  The content is
pruned from the Fedora project's landing page.  We can then use this to
point our irc channels to as a way to notify users.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Review by rhatdan — APPROVED** [review]
LGTM

File: community/irc.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+layout: default
+title: IRC communications
+---
+
+![Podman logo](../images/podman.svg)
+
+# {{ page.title }}
+
+The Podman and many other container projects have recently moved to Libera.Chat
+for IRC communications.  The #podman channel on freenode is no longer being used
```
Comment (TomSweeneyRedHat) [inline community/irc.md]:
```suggestion
for IRC communications.  The #podman channel on the freenode network is no longer being used by
```

**Review by TomSweeneyRedHat — COMMENTED** [review]
Would be good to have a link to this from the community page too.  Perhaps on that page at the end of the "Slack and IRC" section, `Click [here](./irc.md) for more information on the #podman IRC channel.`

**Comment by baude** [comment]
fixed up

**Comment by TomSweeneyRedHat** [comment]
Merging

**Pull request merged**